common/source/docs/common-ac2_simple_geofence.rst

```diff
@@ -51,19 +52,25 @@ The Fence can be set-up by doing the following:
 -  Set "Max Radius" to the maximum distance from home you want (in
    meters).  This should normally be at least 50m
 
-Enabling the fence with Channel 7 or 8
-======================================
+Enabling the fence with an RC Channel Auxiliary Switch
+======================================================
 
 It is not necessary to set-up a switch to enable or disable the fence
 but if you wish to control the fence with a switch please follow these
 steps:
 
+for firmware versions before 4.0:
+
 -  Go to the Mission Planner's Config/Tuning > Extended Tuning screen and set
    either "Ch7 Opt" OR "Ch8 Opt" to Fence.
+   
+for firmware versions 4.0 or later:
```

Comment:
With the latest MP the auxiliary switch drop-downs work with 4.0 so I think we can just provide the two methods (use MP's extended tuning page, or set the parameters directly).

For the direct-parameter method the users can set CHx_OPT or RCx_OPTION.

Reply:
done